plugins/catalog/src/components/EntityContextMenu/EntityContextMenu.tsx

```diff
@@ -84,6 +86,24 @@ export function EntityContextMenu(props: EntityContextMenuProps) {
     setAnchorEl(undefined);
   };
 
+  const [state, setState] = useState({
+    open: false,
+    vertical: 'top',
+    horizontal: 'right',
+  });
+
+  const { open, vertical, horizontal } = state;
+
+  const handleClose = () => {
+    setState({ ...state, open: false });
+  };
+
+  const copyToClipboard = () => {
+    navigator.clipboard
+      .writeText(window.location.toString())
+      .then(() => setState({ ...state, open: true }));
```

Comment:
shouldn't this be open: false?

Reply:
No, it has to be true to render the Snackbar component when the copy button is clicked.